hkmc2/shared/src/main/scala/hkmc2/codegen/js/JSBuilder.scala

```diff
@@ -22,7 +23,7 @@ abstract class CodeBuilder:
   type Context
   
 
-class JSBuilder(using TL, State, Ctx) extends CodeBuilder:
+class JSBuilder(var internalSymbols: Opt[SeqMap[Str, (Symbol, Str)]] = N)(using TL, State, Ctx) extends CodeBuilder:
```

Comment:
I don't know how many time I'll have to say it. Again, and again, and again, don't put default arguments in core logic!

Reply:
Fixed in 04bacf5.